skfda/datasets/_samples_generators.py

```diff
@@ -481,7 +667,7 @@ def make_random_warping(
     np.square(v, out=v)
 
     # Creation of FDataGrid in the corresponding domain
-    data_matrix = scipy.integrate.cumulative_trapezoid(
+    data_matrix = scipy.integrate.cumtrapz(
```

Comment:
Why did you change the name here? `cumulative_trapezoid` is the modern name, and `cumtrapz` is deprecated.